db/pf-schema-X.Y.Z.sql

```diff
@@ -1261,3 +1255,21 @@ CREATE TABLE auth_log (
   KEY pid (pid),
   KEY  attempted_at (attempted_at)
 ) ENGINE=InnoDB;
+
+--
+-- Creating chi_cache table
+--
+
+CREATE TABLE `chi_cache` (
+  `key` VARCHAR(767),
+  `value` LONGTEXT,
```

Comment:
Should be LONGBLOB not LONGTEXT.
Since we are storing binary data attempting to store it as text could cause issues.